src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -297,7 +298,9 @@
   []
   (map->Query {:projections {"logs" {:type :json
                                      :queryable? false
-                                     :field :reports.logs}
+                                     :field (h/coalesce :logs
```

Comment:
For HyperSQL I guess this hinges on there always being a value in the logs column? What happens in HyperSQL if logs in null?